src/org/labkey/remoteapi/domain/GetDomainCommand.java

```diff
@@ -1,13 +1,10 @@
 package org.labkey.remoteapi.domain;
 
 import org.json.simple.JSONObject;
-import org.labkey.remoteapi.PostCommand;
 
-public class GetDomainCommand  extends PostCommand<DomainResponse>
+public class GetDomainCommand  extends DomainCommand
```

Comment:
I'd leave `GetDomainCommand` an independent class. `getDomain` doesn't require a POST and many of the methods in `DomainCommand` aren't relevant (e.g. `setDomainKind`).

Reply:
Done!